release-1.8/release_notes_draft.md

```diff
@@ -221,3 +221,24 @@ kind.
 * [alpha] Support ipvs mode for kube-proxy([#46580](https://github.com/kubernetes/kubernetes/pull/46580), [@haibinxie](https://github.com/haibinxie))
 
 ## External Dependencies
+Continuous integration builds have used the following versions of external
+dependencies, however, this is not a strong recommendation and users should
+consult an appropriate installation or upgrade guide before deciding what
+versions of etcd, docker or rkt to use.
+
+- Docker versions 1.11.2, 1.12.6, 1.13.1 (17.03.2) have been validated
+    - Docker version 1.13.1 (17.03.2) [#42926](https://github.com/kubernetes/kubernetes/issues/42926)
+        - Shared PID namespace, live-restore and overlay2 are supported.
```

Comment:
remove this statement or restate like: "Share PID namespace, .... are validated"?

Reply:
Also we normally only include the known issue for each docker release we found during the qualification process. I think there is no need to have above sentence here.